### PR TITLE
move from runtimeOnly to implementation

### PR DESCRIPTION
**Pull request by snesm**
# BACKEND PULL REQUEST
Move Azure dependencies (with minor updates)  from `runtimeOnly` to `implementation`
and eliminate update conflict between:
`  implementation 'com.azure:azure-storage-queue:12.12.+'
`and
`  runtimeOnly "com.azure:azure-storage-queue:12.12.1"
`
## Related Issue

- Why is this being done? Link to issue, or a few sentences describing why this PR exists

## Changes Proposed

- This removes a build time directive conflict that appears as azure-storage-queue versions change.
- This converts the `runtimeOnly` directive to `implementation` (which includes runtimeOnly).

## Additional Information

- decisions that were made
- notice of future work that needs to be done

## Testing

- How should reviewers verify this PR?

## Checklist for Primary Reviewer

- [x] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke tested
- [ ] Any content updates (user-facing error messages, etc) have been approved by content team
- [ ] Any changes that might generate questions in the support inbox have been flagged to the support team
- [ ] GraphQL schema changes are backward compatible with older version of the front-end
- [ ] Changes comply with the SimpleReport Style Guide
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

----

### CODE REVIEW

File: backend/build.gradle [hunk 1]
```diff
@@ -105,10 +108,6 @@ dependencies {
     testCompile "com.github.tomakehurst:wiremock-jre8:2.28.1"
 
     // Forcing these version resolutions over the BOM of spring-cloud-dependencies below
```
Comment (rin-skylight) [inline backend/build.gradle:110]: Can this comment on line 110 be moved to follow the relocated lines, or purged if no longer needed?

**Review by rin-skylight — CHANGES_REQUESTED** [review]

**Comment by rin-skylight** [comment]
This change was smoke tested on `pentest`, and the system appears to operate normally! Everything functionally looks good.

File: backend/build.gradle (hunk 1 above)
Comment (snesm) [reply backend/build.gradle:110]: good point, might be misleading

**Review by rin-skylight — DISMISSED** [review]
Looks great! Thanks for taking care of the feedback. ^-^

**Review by alismx — APPROVED** [review]
lgtm

**Review by rin-skylight — APPROVED** [review]

**Pull request merged**